### PR TITLE
Weibull distribution terms added

### PR DESCRIPTION
**Pull request by aafolabi**
Description  
-----------------------------------------------------
Remote sensing trait associated with Weibull probability distribution added.

Checklist 
---------------------------------------------------------------------------------------
- [ ] An OBO file generated from an OWL file
- [ ] New terms
  - [ ] The relevant issue has been closed.
  - [ ] Further work is required.
- [ ] Term updates
  - [ ] The relevant issue has been closed.
  - [ ] Further work is required.
- [ ] CHECKS
  - [ ] New variables have a parent trait ID 
    - [ ] New variables have methods and scales 
  - [ ] The OBO file has been validated
    - [ ] checked in cassavabase
    - [ ] checked CropOntology

### CODE REVIEW

File: cassava_trait.obo [hunk 1]
```diff
@@ -6169,6 +6169,44 @@ xref: CO:curator
...
+name: Weibull shape
+namespace: cassava_trait
+def: "Parameter of Weibull function fitted to the pixel height density distribution of a plot." []
+xref: CO:curator
+is_a: CO_334:0002029 ! Remote sensing trait
+
+[Term]
+id: CO_334:0002036
+name: weibull fit scale
+namespace: cassava_trait
```
[the remark this answers is not in the document]
Comment (aafolabi) [reply cassava_trait.obo]: No, it's the name of the variable.
 The variable has no scale of measurement. It's unitless

File: cassava_trait.obo [hunk 2]
```diff
@@ -6169,6 +6169,44 @@ xref: CO:curator
...
+def: "Parameter of Weibull function fitted to the pixel height density distribution of a plot. Weibull fit is described as : y = b/a* (x/a)^b-1 * exp(-(x/a)^b)" []
+synonym: "wscale" EXACT []
+xref: Co:curator
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0002034 ! Weibull scale
+relationship: variable_of CO_334:0010496 ! Calculation: Weibull function
+
+[Term]
+id: CO_334:0002037
+name: weibull fit shape
```
[the remark this answers is not in the document]
Comment (aafolabi) [reply cassava_trait.obo]: Yes. Thats the name

File: cassava_trait.obo [hunk 3]
```diff
@@ -6169,6 +6169,44 @@ xref: CO:curator
...
+id: CO_334:0002034
+name: Weibull scale
+namespace: cassava_trait
+def: "Parameter of weibull function fitted to the pixel height density distribution of a plot." []
+xref: CO:curator
+is_a: CO_334:0002029 ! Remote sensing trait
+
+[Term]
+id: CO_334:0002035
+name: Weibull shape
```
[the remark this answers is not in the document]
Comment (aafolabi) [reply cassava_trait.obo]: Yes. Weibull shape and Weibull scale trait

File: cassava_trait.obo (hunk 2 above)
[the remark this answers is not in the document]
Comment (nmenda) [reply cassava_trait.obo]: do you need to change the name and add to it 'variable' ?
"weibull fit shape variable"  ?

**Pull request merged**